src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -86,6 +89,8 @@ internal static IntPtr LoadNativeLibrary()
                 }
 #endif
 
+                didLoadNativeLibrary = lib != IntPtr.Zero;
```

Comment:
This does not need to be global static. It can be set locally when DllNotFoundException is caught below.

Reply:
OK - I updated the code to avoid this variable alltogether.